CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ $ git clone --recurse-submodules -j8 git@github.com:sablier-labs/v2-core.git
 Then, inside the project's directory, run this to install the Node.js dependencies:
 
 ```shell
-$ pnpm install
+$ bun install
 ```
 
```

Comment:
We should also add the following to the contribution guide:

To build the project:
```bash
$ bun run build
``` 

To see a list of all available scripts:
```bash
$ bun run
```

Reply:
Good idea